include/inviwo/core/datastructures/spatialdata.h

```diff
@@ -316,6 +318,20 @@ StructuredGridEntity<N>::StructuredGridEntity(const Matrix<N + 1, float>& modelM
                                               const Matrix<N + 1, float>& worldMatrix)
     : SpatialEntity<N>(modelMatrix, worldMatrix) {}
 
+template <unsigned int N>
+Vector<N, float> StructuredGridEntity<N>::getSpacing() const {
+    const auto dimensions = getDimensions();
+    const auto basis = this->getBasis();
+
+    Vector<N, float> spacing;
+
+    for (unsigned int i{0}; i < N; ++i) {
+        spacing[i] = glm::length(basis[i]) / dimensions[i];
```

Comment:
Shouldn't this be dividing by `dims-1`

Reply:
I was wondering, too. The thing is that in the constructor for `StructuredGridEntity` that takes in spacing as a parameter we do 
```c++
for (unsigned int i = 0; i < N; ++i) {
    basis[i][i] = dimensions[i] * spacing[i];
}
```
so I simply reversed that operation.

I assume it depends on how you define your dimensions, vertex-based or voxel-based.